lib/src/base-styles/_nx-card.scss

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-card-container {
+  display: flex;
+  margin: $nx-spacing-md 0 $nx-spacing-l 0;
+}
+
+.nx-card {
+  background-color: #fff;
+  border-radius: $nx-border-radius;
+  box-sizing: border-box;
+  padding: $nx-spacing-l $nx-spacing-xxl;
+}
+
+.nx-card__call-out {
+  .nx-icon {
+    font-size: 48px;
+  }
+
+  .nx-icon--large {
+    font-size: 64px;
+  }
+}
+
+.nx-call__call-out-text {
+  @include bold();
+
+  font-size: 32px;
+}
+
+// Vertically oriented cards appear in a row container
+.nx-card-container--row {
+  @include container-horizontal;
+
+  .nx-card {
+    display: flex;
+    flex-direction: column;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    margin: 0 $nx-spacing-md 0 0;
```

Comment:
This margin could cause problems if someone wants something like rows of 3 cards in a container that is exactly wide enough for three cards plus their interior margins.  I'd consider using `column-gap` here instead of margin.

Reply:
> This margin could cause problems if someone wants something like rows of 3 cards in a container that is exactly wide enough for three cards plus their interior margins. I'd consider using `column-gap` here instead of margin.

I didn't do this because IE11 doesn't support `column-gap`.